From e0d14aff112f2def0907c538fb5a0215a1a2523e Mon Sep 17 00:00:00 2001
From: Ankush Pathak <ankush.pathak@chainguard.dev>
Date: Mon, 22 Sep 2025 10:00:30 +0000
Subject: [PATCH] Pin net.java.dev.jna:jna to 5.13 release instead of the
 SNAPSHOT.

Chainguard Java repos do not maintain SNAPSHOTs only release versions.
---
 pom.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/pom.xml b/pom.xml
index 33e44c7f..36f9fe7b 100644
--- a/pom.xml
+++ b/pom.xml
@@ -640,6 +640,13 @@
                 <configuration>
                     <layout>ZIP</layout>
                 </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>net.java.dev.jna</groupId>
+                        <artifactId>jna</artifactId>
+                        <version>5.13.0</version>
+                    </dependency>
+                </dependencies>
             </plugin>
 
             <plugin>
-- 
2.51.0

